ci-operator/step-registry/fips-check/fips-check-ref.yaml

```diff
@@ -0,0 +1,12 @@
+ref:
+  as: fips-check
+  from: cli
+  commands: fips-check-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: FIPS_ENABLED
+  documentation: >-
+    This step checks if FIPS is enabled in all nodes of the cluster and exit accordingly depending on what value the environment variable ${FIPS_ENABLED} holds.
```

Comment:
Just wondering - should this step be done pretty much everywhere, at least in the workflows that run the shared e2e testsuite?